### PR TITLE
Add CSU SA register settings for i.MX6UL

### PR DESCRIPTION
**Pull request by Emantor**
This pr configures the secure access register of the i.MX6 CSU to set almost all non-TrustZone aware masters to non-secure and locks the settings afterwards.

### CODE REVIEW

**Comment by Emantor** [comment]
@clementfaure @MrVan This requires a plat-imx maintainer review. Can you also confirm that the `CSU SA` register for i.MX7 is at the same offset as the i.MX6 variant? Than I can add it to the register file to make shippable happy.

**Comment by clementfaure** [comment]
Hi @Emantor,
the offset for ```CSU_SA``` is ```0x218``` on i.MX7D

**Comment by Emantor** [comment]
Thanks, I have force pushed a new version which includes a change to the i.MX7 register file as well.

File: core/arch/arm/plat-imx/drivers/imx_csu.c [hunk 1]
```diff
@@ -52,33 +62,44 @@ const struct csu_setting csu_setting_imx7ds[] = {
 	{(-1), 0},
 };
 
+const struct csu_sa_setting csu_sa_imx6ul = { 0x10554550, 0x20CC8CC2 };
```
Comment (clementfaure) [inline core/arch/arm/plat-imx/drivers/imx_csu.c]: Could you add a comment describing which master has secure access?

**Comment by clementfaure** [comment]
```Reviewed-by: Clement Faure <clement.faure@nxp.com>```

**Comment by Emantor** [comment]
Tag applied and force pushed.

**Pull request merged**